### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by C6OI** [community/community#146674] [New to GitHub]
Hello. 
I created ticket #3024868 on September 29. Today it is already 71 days since I created it, but I have not received any response. I don't know what to do, should I create a new ticket or should I try to reach support in another way? 
Thank you in advance for your help.

**Comment by ghostinhershell** [comment]
Hi @C6OI,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
